crates/biome_css_formatter/src/css/auxiliary/empty_declaration.rs

```diff
@@ -0,0 +1,10 @@
+use crate::prelude::*;
+use biome_css_syntax::CssEmptyDeclaration;
+use biome_rowan::AstNode;
+#[derive(Debug, Clone, Default)]
+pub(crate) struct FormatCssEmptyDeclaration;
+impl FormatNodeRule<CssEmptyDeclaration> for FormatCssEmptyDeclaration {
+    fn fmt_fields(&self, node: &CssEmptyDeclaration, f: &mut CssFormatter) -> FormatResult<()> {
+        format_verbatim_node(node.syntax()).fmt(f)
```

Comment:
This must be implemented

Reply:
sorry, i forget here, i implement this method and add a test case for this at `declaration.css` test file